.github/workflows/_linux-test.yml

```diff
@@ -176,7 +176,7 @@ jobs:
           github-token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Stop monitoring script
-        if: always() && ${{ steps.monitor-script.outputs.monitor-script-pid }}
+        if: always() && steps.monitor-script.outputs.monitor-script-pid
```

Comment:
I guess this would also work:
`${{ always() && steps.monitor-script.outputs.monitor-script-pid }}`

But, as Github says, "you may omit the expression syntax (${{ }}) because GitHub automatically evaluates the if conditional as an expression" - https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution